test/index.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+var assert = require('assert');
+
+var Z = require('sanctuary-type-classes');
```

Comment:
This will introduce a circular dependency, which is concerning. At least it'll be a circular *dev* dependency. Does anyone know whether this will cause problems?